engine/cld/config/config.go

```diff
@@ -23,6 +24,10 @@ type Config struct {
 	// and deployment settings. This configuration varies between environments
 	// (development, staging, production) and contains sensitive data.
 	Env *cfgenv.Config
+
+	// Jira contains JIRA integration configuration including connection details
+	// and field mappings for using JIRA in resolvers.
+	Jira *cfgjira.JiraConfig
```

Comment:
Following the above pattern `cfgenv.Config`, should  we just call JiraConfig  JIRA, eg `cfgjira.Config` ?
```suggestion
	Jira *cfgjira.Config
```